### PR TITLE
* Added Amazon Corretto Java 8 Updated 232

### PR DESCRIPTION
**Pull request by philiplourandos**
* Added Amazon Corretto Java 11 Update 5

### CODE REVIEW

**Comment by philiplourandos** [comment]
Hi Marco,

Are you refering to the '8.0.202'. There is the removal for 11.0.4 and 8.0.222

**Comment by marc0der** [comment]
I see that we have multiple Corretto migrations, so that would mean migrations 108, 109, 172, 173. I see you already have the version deletions in your migrations so ignore the last part.

File: src/main/scala/io/sdkman/changelogs/JavaMigrations.scala [hunk 1]
```diff
@@ -701,4 +701,24 @@ class JavaMigrations {
...
+  }
+
+  @ChangeSet(order = "204", id = "204-add_corretto_java11_update_5", author = "philiplourandos")
+  def migrate204(implicit db: MongoDatabase) = {
+    List(
+      Version("java", "11.0.5-amzn", "https://d3pxv6yz143wms.cloudfront.net/11.0.5.10.1/amazon-corretto-11.0.5.10.1-linux-x64.tar.gz", Linux64, Some(Amazon)),
+      Version("java", "11.0.5-amzn", "https://d3pxv6yz143wms.cloudfront.net/11.0.5.10.1/amazon-corretto-11.0.5.10.1-macosx-x64.tar.gz", MacOSX, Some(Amazon)),
+      Version("java", "11.0.5-amzn", "https://d3pxv6yz143wms.cloudfront.net/11.0.5.10.1/amazon-corretto-11.0.5.10.1-windows-x64.zip", Windows, Some(Amazon))
+    )
+    Seq(Linux64, MacOSX, Windows).foreach(platform => removeVersion(candidate = "java", version = "1.0.4-amzn", platform))
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/JavaMigrations.scala]: Should this be removing `11.0.4-amzn`?

File: src/main/scala/io/sdkman/changelogs/JavaMigrations.scala [hunk 2]
```diff
@@ -701,4 +701,24 @@ class JavaMigrations {
...
+    Seq(Linux64, MacOSX, Windows).foreach(platform => removeVersion(candidate = "java", version = "8.0.222-amzn", platform))
+  }
+
+  @ChangeSet(order = "204", id = "204-add_corretto_java11_update_5", author = "philiplourandos")
+  def migrate204(implicit db: MongoDatabase) = {
+    List(
+      Version("java", "11.0.5-amzn", "https://d3pxv6yz143wms.cloudfront.net/11.0.5.10.1/amazon-corretto-11.0.5.10.1-linux-x64.tar.gz", Linux64, Some(Amazon)),
+      Version("java", "11.0.5-amzn", "https://d3pxv6yz143wms.cloudfront.net/11.0.5.10.1/amazon-corretto-11.0.5.10.1-macosx-x64.tar.gz", MacOSX, Some(Amazon)),
+      Version("java", "11.0.5-amzn", "https://d3pxv6yz143wms.cloudfront.net/11.0.5.10.1/amazon-corretto-11.0.5.10.1-windows-x64.zip", Windows, Some(Amazon))
+    )
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/JavaMigrations.scala]: I guess we need to validate and insert these too.

File: src/main/scala/io/sdkman/changelogs/JavaMigrations.scala (hunk 1 above)
Comment (philiplourandos) [reply src/main/scala/io/sdkman/changelogs/JavaMigrations.scala]: *faceplam* apologies for that

**Comment by marc0der** [comment]
I made some minor changes and then did a [squash of commits](https://github.com/sdkman/sdkman-db-migrations/commit/1facf8e49204de92abcdbd8102eae42b720bc95f) locally. I've pushed your changes so they should appear now.

Looks like the Github UI got horribly confused about what I did though, so closing this issue manually :laughing:

**Pull request closed without merging**